### ISSUE TITLE
Nautilus ThrashBar missing background-color SOLVED

### ISSUE THREAD

**Issue by Sadi58** [shimmerproject/Greybird#82]
I've just discovered how to restore the missing background-color in Nautilus Trash window's infobar at the top under Ubuntu 14.10 (though I could't draw a bottom border) which requires adding the following lines at the end of the Nautilus section in "gnome-applications.css" file. Sorry, I don't know how to create a patch here yet:
```   
NautilusTrashBar {
    background-color: @info_bg_color;
    color: @info_fg_color;
}
```

**Comment by ochosi** [comment]
Hm, I don't use Nautilus, so I can't even test this.
@satya164: Thoughts?

**Comment by Sadi58** [comment]
For your information I've also made a bug report about this here: https://bugs.launchpad.net/ubuntu/+source/nautilus/+bug/1440354

**Comment by Sadi58** [comment]
I can confirm that I've successfully tested and updated the code above as:
```
NautilusTrashBar {
    background-color: @info_bg_color;
    color: @info_fg_color;
    border-color: #d6d2a0;
    border-style: solid;
    border-width: 0 0 1px 0; }
}
```

**Comment by Sadi58** [comment]
This problem is solved in Ubuntu 15.04.
